### ISSUE TITLE
resthooks does not expose customer sunscriptions

### ISSUE THREAD

**Issue by kitoban** [reapit/foundations#4755]
**Describe the bug**
resthooks api does not provide all the necessary data to configure webhooks

**To Reproduce**
Steps to reproduce the behaviour eg:
1. query data structure for a webhook
2. attempt to update the webhook structure

**Expected behaviour**
All fields within the ui to be configurable on the api
![image](https://user-images.githubusercontent.com/5329503/128993805-b06e1041-31e9-429b-9a14-ac2910b1e33d.png)
Subscription customers is missing, this is the field that we would want to be able to configure on the api.
Our use case is to be able to add/remove customers from our webhooks as we bring customers online within our systems

**Additional context**
All other settings on the webhooks have been manually setup, the only thing we would need this API to do is configuring the customers on the webhooks.

**Comment by cbryanreapit** [comment]
@kitoban this is by design. Access tokens are assigned to a specific customers data, so we enforce that webhooks can only be added for that specific customer using the RESTHooks API. You can either create a single webhook in the UI that will listen to all customer events, or you can create a per customer subscription using the RESTHooks API.

**Comment by kitoban** [comment]
Are you able to associate the same installed webhook to different customer ID's? 
(Not something I can test on the SBOX as it only exposes a single customer)

**Comment by cbryanreapit** [comment]
Yes - you can have a customer specific hook for the same set of topics. You should consider each POST payload to RESTHooks as a completely standalone subscription

**Comment by kitoban** [comment]
Ok, can work with that. although I suspect you admin ui might not work so well once we have a number of these setup, as I would suspect this could result in a large number of subscriptions, and there is currently no way to filter on the web console by customer.
